lib/public/views/QcFlags/details/qcFlagDetailsComponent.js

```diff
@@ -105,7 +105,12 @@ export const qcFlagDetailsComponenet = (qcFlagDetailsModel, additionalDetailsCon
                     Loading: () => deleteButton(null, 'Processing...'),
                     Success: () => deleteButton(null, 'Processed!'),
                     Failure: () => deleteButton(() => qcFlagDetailsModel.delete()),
-                    NotAsked: () => deleteButton(() => qcFlagDetailsModel.delete()),
+                    NotAsked: () => remoteQcFlag.match({
+                        Success: (qcFlag) => qcFlag.verifications?.length > 0
+                            ? tooltip(deleteButton(), 'Cannot delete verified flag')
```

Comment:
Message is not descriptive enough. I would recommend something along the line:
`Deleting a verified flag is not allowed`

Reply:
applied